common/containers/Tabs/SignAndVerifyMessage/components/SignMessage/index.tsx

```diff
@@ -74,12 +75,16 @@ export class SignMessage extends Component<Props, State> {
               onSignMessage={this.onSignMessage}
             />
 
-            {!!signedMessage && (
+            {signedMessage && (
               <div className="input-group-wrapper SignMessage-inputBox">
                 <label className="input-group">
                   <div className="input-group-header">{translate('MSG_SIGNATURE')}</div>
                   <CodeBlock className="SignMessage-inputBox">
-                    {JSON.stringify(signedMessage, null, 2)}
+                    {JSON.stringify(
+                      { ...signedMessage, address: toChecksumAddress(signedMessage.address) },
```

Comment:
Prefer checksumming in `SignMessageButton.handleSignMessage` so that any implementation of it doesn't need to do this.

Reply:
Agreed